scripts/west_commands/tests/test_nrfjprog.py

```diff
@@ -473,7 +473,7 @@ def test_nrfjprog_create(check_call, get_snr, require, test_case,
     if test_case.softreset:
         args.append('--softreset')
     if test_case.snr:
-        args.extend(['--snr', TEST_OVR_SNR])
+        args.extend(['--dev-id', TEST_OVR_SNR])
```

Comment:
should update all the other references to `--snr` in this file.

Reply:
`nrfjprog` uses `--snr`.

Reply:
Hmm, after checking I'm not sure removing SNR improves the code, as nrfjprog uses --snr and we're testing nrfjprog is run with the correct serial number, obtained from dev-id.